apps/glusterfs/app.go

```diff
@@ -233,6 +247,25 @@ func (a *App) Upgrade(tx *bolt.Tx) error {
 	return nil
 }
 
+func (a *App) setFromEnvironmentalVariable() {
+	var err error
+	env := os.Getenv("HEKETI_AUTO_CREATE_BLOCK_HOSTING_VOLUME")
+	if "" != env {
+		a.conf.CreateBlockHostingVolumes, err = strconv.ParseBool(env)
+		if err != nil {
+			logger.LogError("Error: Parse bool in Create Block Hosting Volumes: %v", err)
+		}
+	}
+
+	env = os.Getenv("HEKETI_NEW_BLOCK_HOSTING_VOLUME_SIZE")
+	if "" != env {
+		a.conf.NewBlockHostingVolumeSize, err = strconv.Atoi(env)
```

Comment:
Added correctness. If given an invalid value it will fall back to default values.